tests/integration/test_snakemake.py

```diff
@@ -27,14 +27,16 @@ def test_missing_results(mocker, caplog):
         new_callable=mocker.PropertyMock,
         return_value="sleep 1s",
     )
-    with tempfile.TemporaryDirectory(dir="tests/integration/") as results_dir:
-        results_dir = Path(results_dir).resolve()
-        with pytest.raises(SystemExit) as exit:
-            main(
-                SPECIFICATIONS_DIR / "integration" / "pipeline.yaml",
-                SPECIFICATIONS_DIR / "input_data.yaml",
-                SPECIFICATIONS_DIR / "environment_local.yaml",
-                str(results_dir),
-            )
-        assert exit.value.code == 1
-        assert "MissingOutputException" in caplog.text
+    results_dir = tempfile.mkdtemp(dir=RESULTS_DIR)
```

Comment:
changed the temp directory to be consistent with e2e tests (which is needed for proper jenkins build)